src/uhs/twophase/locking_shard/interface.hpp

```diff
@@ -15,12 +16,8 @@
 namespace cbdc::locking_shard {
     /// Transaction type processed by locking shards.
     struct tx {
-        /// The TX ID of the transaction, if provided.
-        std::optional<hash_t> m_tx_id{};
-        /// Vector of input hashes for the shard to process as spent.
-        std::vector<hash_t> m_spending{};
-        /// Vector of output hashes to create on the shard.
-        std::vector<hash_t> m_creating{};
+        /// Compact TX.
```

Comment:
Is there a point in keeping this data structure if it's reduced to just contain a `compact_tx`? Is it just to avoid a greater delta for the PR?

Reply:
It avoids a greater delta, and leaves the structure available for future expansion for other tamper detection features.